README.md

```diff
@@ -8,15 +8,28 @@
 # aries-framework-go
 
 ## Introduction
-A go framework for Aries
+A Go framework for Aries
 
-## Contributing
+## Build
+##### Prerequisites for test
+- Go 1.12
```

Comment:
Of course, also Go 1.12 is also a general requirement :).

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/4114ed97adddd4bae7ca7aa37395dd5030ab4c59..bc1bec9caa24a8ee97ec09d6762fe66a2e5bdefb#diff-04c6e90faac2675aa89e2176d2eec7d8R14).